packages/pigeon/example/README.md

```diff
@@ -284,7 +284,7 @@ private class PigeonFlutterApi {
     aString aStringArg: String?, completion: @escaping (Result<String, Error>) -> Void
   ) {
     flutterAPI.flutterMethod(aString: aStringArg) {
-      completion(.success($0))
+      completion(.success(try! $0.get()))
```

Comment:
Code didn't compile until I fixed this. I'm assuming our CI doesn't verify the example compiles. Is this unintentional?

I also had to fix the same line for Kotlin.

Reply:
Just ran into this in my pr as well. Not sure why the examples are getting compiled in ci though

Reply:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8731158450509496993/+/u/Run_package_tests/build_examples/stdout

```
============================================================
|| Running for packages/pigeon [@0:00]
============================================================

Building for: Android

Skipping Android for pigeon/example; not supported.
SKIPPING: No examples found supporting requested platform(s).
```

O.o

Let me look into what's going on here.

Reply:
Ah, it's because the actual example is `example/app/`, not `example/`, but `example.pubspec.yaml` exists so the tooling sees that `example` is a package.

I'll play with fixing this in a separate PR. I think the pubspec may have only been there for the old excerpt system.